XMonad/Actions/DynamicProjects.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE DeriveDataTypeable #-}
+
```

Comment:
It would be nice if these kinds of things were just the removal of a line and didn't introduce an empty line as well